src/core/chatnet-rec.h

```diff
@@ -4,6 +4,7 @@ int chat_type; /* chat_protocol_lookup(xx) */
 char *name;
 
 char *nick;
+char *alternate_nick;
```

Comment:
This is never freed in `chatnet_destroy` nor is saved/loaded in `chatnet_config_save`/`chatnet_read`.